src/api/manifest/ManifestEditor.ts

```diff
@@ -28,15 +29,17 @@ export class ManifestEditor {
     }
   }
 
-  public static readAndParseManifest(path: string) {
+  public static async readAndParseManifest(path: string) {
     try {
-      return readJson(path)
+      return await readJson(path)
```

Comment:
Wow, I think I was the responsible for this bug haha - I really learned a lot in the past couple months ,thanks for solving this